addons/purchase_requisition/wizard/purchase_requisition_create_alternative.py

```diff
@@ -78,5 +78,9 @@ def _get_alternative_values(self):
             'dest_address_id': self.origin_po_id.dest_address_id.id,
         }
         if self.copy_products and self.origin_po_id:
-            vals['order_line'] = [Command.create({'product_id': line.product_id.id, 'product_qty': line.product_qty}) for line in self.origin_po_id.order_line]
+            vals['order_line'] = [Command.create(self._get_alternative_line_value(line)) for line in self.origin_po_id.order_line]
```

Comment:
To check if it's possible to use a copy_data. It would avoid extra module like this one